ksql-functional-tests/src/main/java/io/confluent/ksql/test/model/TestCaseNode.java

```diff
@@ -78,7 +78,7 @@
   private final Optional<PostConditionsNode> postConditions;
 
   // CHECKSTYLE_RULES.OFF: CyclomaticComplexity|NPathComplexity
-  TestCaseNode(
+  public TestCaseNode(
```

Comment:
We need to update the exceptions thrown in this constructor below. The exception currently thrown if an input file doesn't have the correct structure isn't informative of the error:
```
Victoria-Xias-MBP13:ksql victoria.xia$ ./bin/ksql-test-runner -s statements.sql -o output.txt -i output.txt
Exception in thread "main" io.confluent.ksql.test.tools.exceptions.InvalidFieldException: inputs and outputs: either both, or neither, field should be set
	at io.confluent.ksql.test.model.TestCaseNode.<init>(TestCaseNode.java:112)
	at io.confluent.ksql.test.tools.KsqlTestingTool.runWithTripleFiles(KsqlTestingTool.java:106)
	at io.confluent.ksql.test.tools.KsqlTestingTool.main(KsqlTestingTool.java:56)
```

Reply:
Good point. Added a better message along with test for it.

Reply:
Has this been addressed? I still get 
```
Victoria-Xias-MBP13:ksql victoria.xia$ ./bin/ksql-test-runner -i input.json -o output.json -s statements.sql
Exception in thread "main" io.confluent.ksql.test.tools.exceptions.InvalidFieldException: inputs and outputs: either both, or neither, field should be set
	at io.confluent.ksql.test.model.TestCaseNode.<init>(TestCaseNode.java:112)
	at io.confluent.ksql.test.tools.KsqlTestingTool.runWithTripleFiles(KsqlTestingTool.java:117)
	at io.confluent.ksql.test.tools.KsqlTestingTool.main(KsqlTestingTool.java:56)
```
when I pass in an output file that looks like 
```
{
  "output": [
    {"topic": "test_topic", "key": 0, "value": "0,zero,0.0"},
    {"topic": "test_topic", "key": 100, "value": "100,100,0.0"},
    {"topic": "test_topic", "key": 101, "value": "101,101,0.0"}
  ]
}
```
(where you'll notice the file has `output` instead of `outputs`.

Reply:
The same error also appears when I pass in an output file that looks like
```
{
   "outputs": [
   ]
}
```
which seems like a bug, since it should be valid to expect no output records.